src/utils/CharactersPageUtils.js

```diff
@@ -0,0 +1,86 @@
+import { nanoid } from "@reduxjs/toolkit"
+
+export const prepareCharacterForFormReset = ({
+  name,
+  eyeColor,
+  height,
+  gender,
+  birthYear,
+  homeworld,
+  species,
+  filmConnection,
+}) => {
+  return {
+    name,
+    eyeColor,
+    height: height || "",
+    gender:
+      gender !== "n/a"
+        ? {
+            value: gender,
+            label: gender,
+          }
+        : null,
+    birthYear: birthYear !== "unknown" ? birthYear : "",
+    homeworld: homeworld.name,
+    species: species?.name
+      ? {
+          value: species.name,
+          label: species.name,
+        }
+      : null,
+    numberOfFilms: filmConnection.totalCount,
+  }
+}
+
+export const transformCharacterForSubmit = (
```

Comment:
Please split this function into two separate ones, one handling creation and one handling edit